src/tests.cpp

```diff
@@ -99,22 +99,27 @@ const char* BATCH_JSON = R"({ "type": "SPAT", "timestamp": 1735713156407, "hex":
 //     CHECK(std::strcmp(byte_node.text().get(), BSM_HEX) == 0);
 // }
 
-TEST_CASE("Encode ASD", "[encoding]" ) {
-    std::cout << "=== Encode ASD ===" << std::endl;
+/**
```

Comment:
question: If this test case isn't valid, can it be removed?

Reply:
yeah I think it could be removed; opened https://github.com/Trihydro/asn1_codec/pull/2 for this